.storybook/stories/wizard/wizard.stories.ts

```diff
@@ -108,4 +108,10 @@ const WizardTemplate: Story = args => ({
 
 export const Wizard: StoryObj = {
   render: WizardTemplate,
+  play: removeFocusOutline,
 };
+
+function removeFocusOutline({ canvasElement }: StoryContext) {
+  // remove keyboard focus outline from modal title
+  canvasElement.querySelector<HTMLElement>(':focus')?.blur();
+}
```

Comment:
I see usage of `removeFocusOutline` function in modal as well. Can you move it to helpers so it can be shared between all stories?

Reply:
I'll move it.

Reply:
done: 5aba9e2c44abded87fe1e69c522bef601059ebb9